tests/unit/changes_tests.py

```diff
@@ -465,14 +465,29 @@ def test_get_feed_using_doc_ids(self):
         self.assertSetEqual(set([x['id'] for x in changes]), expected)
         self.assertTrue(str(feed.last_seq).startswith('100'))
 
-    def test_invalid_argument(self):
+    def test_get_feed_with_custom_filter_query_params(self):
         """
-        Test that an invalid argument is caught and an exception is raised
+        Test using feed with custom filter query parameters.
         """
-        feed = Feed(self.db, foo='bar')
-        with self.assertRaises(CloudantArgumentError) as cm:
-            invalid_feed = [x for x in feed]
-        self.assertEqual(str(cm.exception), 'Invalid argument foo')
+        feed = Feed(
+            self.db,
+            filter='mailbox/new_mail',
+            foo='bar',  # query parameters to a custom filter
+            include_docs=False
+        )
+        params = feed._translate(feed._options)
+        self.assertEquals(params['filter'], 'mailbox/new_mail')
+        self.assertEquals(params['foo'], 'bar')
+        self.assertEquals(params['include_docs'], 'false')
+
+    # def test_invalid_argument(self):
```

Comment:
This test is no longer valid as passing `foo='bar'` is now accepted under `ANY_ARG: ANY_TYPE`.